sale_rental/models/sale_order.py

```diff
@@ -0,0 +1,286 @@
+# Copyright 2014-2019 Akretion (http://www.akretion.com)
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+# Copyright 2016-2019 Sodexis (http://sodexis.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools import float_compare
+from dateutil.relativedelta import relativedelta
+import odoo.addons.decimal_precision as dp
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    def action_cancel(self):
+        """
+            In case cancelling a SO which sells rental product.
+            Picking in should be created manually
+        """
+        res = super(SaleOrder, self).action_cancel()
+        for order in self:
+            for line in order.order_line.filtered(
+                    lambda l: l.rental_type == 'rental_extension' and
+                    l.extension_rental_id):
+                initial_end_date = line.extension_rental_id.end_date
+                line.extension_rental_id.in_move_id.write({
+                    'date_expected': initial_end_date,
+                    'date': initial_end_date,
+                    })
+        return res
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    rental = fields.Boolean(string='Rental')
```

Comment:
```suggestion
    rental = fields.Boolean(string='Rental', default=False)
```

Reply:
False is already the default for boolean fields.